packages/protocol/contracts/L1/TaikoData.sol

```diff
@@ -124,8 +124,6 @@ library TaikoData {
         bytes32 signalRoot;
         bytes32 graffiti;
         address prover;
-        uint32 parentGasUsed;
-        uint32 gasUsed;
```

Comment:
seems `TaikoL2.anchor` is still using `parentGasUsed` as one of its parameters, remove this from evidence means its not in circuits public input anymore (since right now the public input is `keccak256(abi.encode(evidence))`), not sure if its ok?

Reply:
Yep that's the reason we stored the gas_used onchain in the fork choice and had to link it with the parent one, because we somehow had to have the gas_used of the previous block available to be able to correctly update the L2 basefee. Using the gas_used of the current block in the anchor transaction would create a circular dependency when creating the block, so we had to avoid that somehow.

But we will now have access to all 256 previous block headers in the circuits, so we can now get the parent gas used from the block header of the previous block in a verifiable way!

So I agree that we can remove this now, though for a different reason than it seems this PR was created haha